bezier/src/lib.rs

```diff
@@ -81,6 +81,17 @@ pub type Point = euclid::Point2D<f32>;
 /// Alias for ```euclid::Point2D<f32>```.
 pub type Vec2 = euclid::Point2D<f32>;
 
+/// Alias for ```euclid::Size2D<f32>```.
+pub type Size = euclid::Size2D<f32>;
+
+/// Alias for ```euclid::Rect<f32>```
+pub type Rect = euclid::Rect<f32>;
+
+/// Allows to get a Rect structure which contains the Segment
+trait HasBoundingRect {
```

Comment:
The idea of adding a trait is interesting, but I'd prefer to do this separately. There are other common APIs between the different types of segments, and I'd like to brainstorm what the use cases are and how these traits should look like first (should we group them in some `Curve` trait? or should we have a trait for each group of common method? etc.).
Also, having a method exposed as trait implementations means you need to import the trait to use it which I find inconvenient, so I prefer the approach of implementing all methods on the type itself and then using them in trait impls.

So, could you remove the trait from this PR, I'll  open an issue so that we can brainstorm the API and we'll add traits as a followup.